src/app.svelte

```diff
@@ -24,9 +24,11 @@
   <div class="applicationcontainer flex flex-col font-body font-medium w-full">
     {#await configPromise then config}
       {#if config.showTestEnvNotification}
-        <Container {...containerStyles.red}>
-          <Disclaimer>{$_('DISCLAIMER_WIP')}</Disclaimer>
-        </Container>
+        <div class="print:hidden">
```

Comment:
Kun komponentteihin ei taida suodaan saada laittaa näitä classeja, niin onko tämä nyt se ilmeisin reitti päästä niitä laittamaan? Vaihtoehtohan olisi upottaa vastaava vipu komponentin sisään.

Reply:
Juu tämä lienee ilmeisin reitti tyylitellä lapsia. Toki tuon varoitusbarin hidettäminen printistä tarkoittaa sitä, että käyttäjät voivat printtailla noita oikean näköisinä kehitys/testiympäristössä. En tiedä onko tässä ongelmaa.

Reply:
Jos halutaan vielä tähän varoitus, niin sen pitäisi olla semmoinen että se ei vaikuta muiden elementtien asetteluun. Varmaan tähän sopisi laittaa tuo `print:absolute` ja muutama muu lisävipu.

Reply:
Nyt tuossa viimeisimmässä commitissa tulee testileima niihin printteihin.